scripts/csv_bench.sh

```diff
@@ -0,0 +1,117 @@
+usage() {
+    cat <<_end_
+Usage: csv-bench.sh --path=PATH [--coreneuron]
+
+Generate CSV file summarising a set of benchmark runs.
+
+Options:
+    --path=PATH      Path containing the output
+    --coreneuron     If the path contains CoreNeuron output.
+_end_
+    exit 1
+}
+
+parse_coreneuron=false
+path=
+
+while [ "$1" != "" ]
+do
+    case $1 in
+        --coreneuron )
+            parse_coreneuron=true
+            ;;
+        --path=* )
+            path="${1#--path=}"
+            ;;
+        --path )
+            shift
+            path=$1
+            ;;
+        * )
+            echo "unknown option '$1'"
+            usage
+    esac
+    shift
+done
+
+if [ "$path" == "" ]
+then
+    usage
+    exit 1
+fi
+
+if [ ! -d "$path" ]
+then
+    echo "error: path \"$path\" does not exist"
+    exit 1
+fi
+
+table_line() {
+    fid="$1"
+    line=
+
+    tts=`awk '/^model-run/ {print $2}' "$fid"`
+    ncell=`awk '/^cell stats/ {print $3}' "$fid"`
+
+    line="$(printf "%9d,%12.3f," $ncell $tts)"
```

Comment:
Quoting here is a bit odd: it's quoting "$(printf " then " $ncell $tts)". It all works, but it doesn't look like that is what was intended. `$()` is implicitly quoted, so the following should be sufficient:
```
line=$(printf %9d,%12.3f $ncell $tts)
```

Reply:
Fixed.